pages/osx/shutdown.md

```diff
@@ -17,3 +17,11 @@
 - Reboot in 5 minutes:
 
 `shutdown -r +{{5}}`
+
+- Power off (halt) at 1:00 pm (Uses 24h clock):
+
+`shutdown -h {{1300}}`
+
+- Reboot on May 10th 2042 at 11:30 am:
```

Comment:
IMHO it would be good to show the input format on the description so that the user does not have to "decode" the example tu understand the command.

Reply:
That's a good point. I've now included it in the description. What do you think about that?

Reply:
Looks good to me.